DISTRIBUTION.md

```
@@ -0,0 +1,392 @@
+Description: CLI application to manage programmable banking cards
+EOF
+
+# Build package
+dpkg-deb --build ipb_0.8.3
+```
+
+### Ubuntu PPA (Personal Package Archive)
+
+For easy distribution and automatic updates, set up a PPA on Launchpad:
+
+1. **Create Launchpad account** at https://launchpad.net
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)